.github/workflows/gempush.yml

```diff
@@ -10,10 +10,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Ruby 2.6
+    - name: Set up Ruby 3.0
```

Comment:
What is the impact of this change on users in terms of ruby version compatibility?

Reply:
the above change is not needed now the issue is fixed by using `bundler 2.4.22` in ci  `Set up Ruby 2.6` was using a higher version of `bundler`.